about.html

```diff
@@ -0,0 +1,443 @@
+                      alt="GitHub Icon"
+                      class="w-8 h-8"
+                    />
+                  </a>
+                </div>
+              </div>
+            </div>
+            <!-- Second card -->
+            <div
+              class="w-full max-w-sm bg-white border border-gray-200 rounded-lg shadow dark:bg-gray-800 dark:border-gray-700 mt-6 md:mt-0"
+            >
+              <div class="p-6">
+                <!-- Card content -->
+                <img
+                  class="w-24 h-24 mb-3 rounded-full shadow-lg mx-auto"
+                  src="assets/images/thasbia.svg"
+                  alt="Profile picture"
+                />
+                <h5
+                  class="mb-1 text-xl font-medium text-gray-900 dark:text-white text-center"
+                >
+                  Thasbia
+                </h5>
+                <span
+                  class="text-sm text-gray-500 dark:text-gray-400 text-center block"
+                  >CyberOps Apprentice</span
+                >
+                <div class="flex justify-center mt-4">
+                  <a
+                    href="https://github.com/yourusername"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    <img
+                      src="assets/icons/github.svg"
+                      alt="GitHub Icon"
+                      class="w-8 h-8"
+                    />
+                  </a>
+                </div>
+              </div>
+            </div>
+            <!-- Third card -->
+            <div
+              class="w-full max-w-sm bg-white border border-gray-200 rounded-lg shadow dark:bg-gray-800 dark:border-gray-700 mt-6 md:mt-0"
+            >
+              <div class="p-6">
+                <!-- Card content -->
+                <img
+                  class="w-24 h-24 mb-3 rounded-full shadow-lg mx-auto"
+                  src="assets/images/nathan.svg"
+                  alt="Profile picture"
+                />
+                <h5
+                  class="mb-1 text-xl font-medium text-gray-900 dark:text-white text-center"
+                >
+                  Nathan
+                </h5>
+                <span
+                  class="text-sm text-gray-500 dark:text-gray-400 text-center block"
+                  >CyberOps Apprentice</span
+                >
+                <div class="flex justify-center mt-4">
+                  <a
+                    href="https://github.com/yourusername"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    <img
+                      src="assets/icons/github.svg"
+                      alt="GitHub Icon"
+                      class="w-8 h-8"
+                    />
+                  </a>
+                </div>
+              </div>
+            </div>
+          </div>
+          <!-- Second row -->
```

Comment:
First row responsiveness on large screens needs changing - currently on large (maybe v.large) screens the 3 cards default to align-left, should stay aligned center.

Reply:
kept as desktop view for now until we refactor at the end when all pages are completed, updated all github usernames